bundle/language_en.properties

```diff
@@ -668,6 +673,107 @@ angal.hospital.visitduration.txt
 angal.hospital.visitendhour.txt                                                                        = Visit end hour
 angal.hospital.visitstarthour.txt                                                                      = Visit start hour
 angal.lab.allnegative.txt                                                                              = All negative.
+angal.invetory.allmedicaladdedsuccessfully.msg                                                         = All medicals are added successfully.
+angal.inventory.allproduct.btn                                                                         = All products
+angal.inventory.askforrealquantityempty.msg                                                            = Do you want to print without the real quantity?
+angal.inventory.canceled                                                                               = Canceled
+angal.inventory.cancelednoteditable.msg                                                                = A canceled inventory can not be modified.
+angal.inventory.cannotcreateanotherinventorywithotherinprogressinthisward.msg                          = Cannot create another inventory with other in progress in this ward.
+angal.inventory.cannotcreateanotherinventorywithstatusdraft.msg                                        = Cannot create another inventory with status draft, Because you already have an inventory with status draft or validated.
```

Comment:
```suggestion
angal.inventory.cannotcreateanotherinventorywithstatusdraft.msg                                        = Cannot create another inventory with status draft, because there exists an inventory with status draft or validated.
```